### DISCUSSION TITLE
TLS certificate reloading?

### DISCUSSION THREAD

**Discussion by tashian** [minio/minio#12863] [Q&A] [answered]
Hi there! I'm using Minio with a [`step-ca`](https://github.com/smallstep/certificates/) internal certificate authority in my homelab. Everything is working great, except when I renew the TLS certificate for Minio, it continues to serve the old certificate. Is there a way to reload the certificate file without restarting Minio?

**Comment by harshavardhana** [comment, marked as answer]
MinIO reloads the cert every 24hours if it's a symbolic link, otherwise, it reloads right away if it's a regular file using filesystem notify framework.
